src/api-doc/api-doc.yaml

```diff
@@ -4436,92 +4439,205 @@ paths:
+                          resolved_by:
+                            type: string
+                            description: ID of the user who resolved the comment, if applicable
+                          resolved_at:
+                            type: string
+                            format: date-time
+                            description: Timestamp when the comment was resolved
+                          context:
+                            type: string
+                            description: Context of the comment (e.g., "page" or any tag)
+                          page:
+                            type: string
+                            description: Page number associated with the comment (e.g., "1")
+                          is_read:
+                            type: boolean
+                            description: Whether the comment has been read
+                          created_at:
+                            type: string
+                            format: date-time
+                            description: Timestamp when the comment was created
+                          updated_at:
+                            type: string
+                            format: date-time
+                            description: Timestamp when the comment was last updated
+                          deleted_at:
+                            type: string
+                            format: date-time
+                            description: Timestamp when the comment was deleted, if applicable
+                          commenter:
+                            type: object
+                            properties:
+                              id:
+                                type: integer
+                                description: ID of the commenter
+                              name:
+                                type: string
+                                description: Name of the commenter
+                          resolver:
+                            type: object
+                            description: Details of the user who resolved the comment, if applicable
             examples:
-              example1:
+              array_of_comments:
                 value:
-                  comment: 'Check spelling'
-                  context: page
-                  page: 1
-                  status: OPEN / DRAFT / RESOLVED
-                  parent_id: 2
-
-      responses:
-        '201':
-          description: Comment updated successfully
-          content:
-            application.json:
-              schema:
-                $schema: http://json-schema.org/draft-04/schema#
-                type: object
-                properties:
-                  responseCode:
-                    type: string
-                  message:
-                    type: string
-                  result:
-                    type: object
+                  comment:
```

Comment:
@adithyadinesh0412  we are not giving any response back they why this is added

Reply:
this is request body example